google/async_resumable_media/_download.py

```diff
@@ -127,10 +128,6 @@ class Download(DownloadBase):
         checksum Optional([str]): The type of checksum to compute to verify
             the integrity of the object. The response headers must contain
             a checksum of the requested type. If the headers lack an
-            appropriate checksum (for instance in the case of transcoded or
```

Comment:
This docstring section still looks like it's being deleted despite previous changes.